native/cocos/math/Mat4.h

```diff
@@ -215,17 +215,6 @@ class CC_DLL Mat4 {
                                   bool isFieldOfViewY = false, float minClipZ = -1, float projectionSignY = 1,
                                   int orientation = 0, Mat4 *dst = nullptr);
 
-    /**
-     * Creates an orthographic projection matrix.
-     *
-     * @param width The width of the view.
-     * @param height The height of the view.
-     * @param zNearPlane The minimum z-value of the view volume.
-     * @param zFarPlane The maximum z-value of the view volume.
-     * @param dst A matrix to store the result in.
-     */
-    static void createOrthographic(float left, float right, float bottom, float top, float zNearPlane, float zFarPlane, Mat4 *dst);
```

Comment:
createOrthogonal？

Reply:
![WXWorkCapture_16784324505912](https://user-images.githubusercontent.com/24495793/224258378-fa61ffa8-da56-4a0f-9e4e-36de9ce43321.png)
orthographic is more proper for projection matrix